bundler/lib/bundler/man/bundle-plugin.1.ronn

```diff
@@ -0,0 +1,61 @@
+bundle-plugin(1) -- Manage Bundler plugins
+==========================================
+
+## SYNOPSIS
+
+`bundle plugin` install PLUGINS [--source=<SOURCE>] [--version=<version>]
+                              [--git|--local_git=<git-url>] [--branch=<branch>] [--ref=<rev>]
+
+`bundle plugin` uninstall PLUGINS
+
+`bundle plugin` list
+
+`bundle plugin` help [COMMAND]
+
+## DESCRIPTION
+
+You can install, uninstall, and list plugin(s) with this command to extend functionalities of Bundler.
+
+## SUB-COMMANDS
+### install
```

Comment:
I know this file is not meant to be read directly, but I think it's customary to add a blank line between headers?

Reply:
Makes sense to me.

Reply:
It doesn't seem applied actually, I meant a blank line between `## SUB-COMMANDS` and `### install`. But this is too minor, let's ignore it, if we want to have this kind of consistency we should use something like https://github.com/markdownlint/markdownlint. Disregard this suggestion.